ui/scss/component/_main.scss

```diff
@@ -101,3 +100,12 @@
 .main--full-width {
   width: 100%;
 }
+
+.main__document-wrapper {
+  width: 60%;
```

Comment:
Are you sure this should be a percentage? If the idea is to narrow for readability, it probably ought to be a fixed size.